### ISSUE TITLE
Move os_lib_model_generation.rb and os_lib_model_simplification.rb to different repo

### ISSUE THREAD

**Issue by DavidGoldwasser** [NREL/openstudio-extension-gem#40]
Confirm that nothing here is using this, and that it is only used by measures in https://github.com/NREL/openstudio-model-articulation. 

@nllong or @kflemin when I move these would I add a `lib/openstudio/extensions/core` or similar directory for resource files only used by measures in one gem.

**Comment by nllong** [comment]
yes, moving the two files to probably the openstudio-model-articulation makes sense.

**Comment by DavidGoldwasser** [comment]
@nllong assuming I put the files in similar structure to how they are here, but maybe not named core? can the measures in the model articulation gem, require the model articulation gem or is that going to create some odd circular logic. One of the urban opt measures has a situation like this and the measure just loads a file external to the measure. That works fine with the measures are used from repo checkout but not stand alone when measures are on BCL. We could copy the resource into the measure but I know we are trying not to do that, so I thought maybe requiring the model articulation gem for the measures within the model articulation gem.

For reference, here is the associated issue in the model articulation gem
https://github.com/NREL/openstudio-model-articulation-gem/issues/16

**Comment by DavidGoldwasser** [comment]
@nllong and @kflemin I was thinking about this issue both because of updating the measure to use the extension gem, as well as conversation with Austin about Alfalfa gem. Moving the resource files make sense, so that the code related to model articulation is with the measures, for those resources that are not more general use, which would still be in extension gem. But the problem starts to be how many gems are we willing to install with OpenStudio, and if we don't then how does a user run this measure (in particular when getting it stand alone from BCL). I had thought of idea of having a way to store something like a gem file in an OSW that would make sure they were installed before run, but that may open up security issue.

Another idea maybe also doesn't make sense but I thought I would suggest it. I know the extension gem can require other gems be installed. Is there a way to have it grab the resources files from the model articulation gem but not the measures. That way the code could be developed across multiple repositories but only the extension gem would need to ship with OpenStudio.

I really don't know the right answer to this so I'm open to your suggestions.
